ui/src/components/header.tsx

```diff
@@ -128,7 +128,7 @@ export const Header = () => {
         </Sheet>
         <Breadcrumb>
           <BreadcrumbList>
-            {organizationMatch && (
+            {organizationMatch && organizationMatch.context && (
```

Comment:
Could we instead use [chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html), like just `(organizationMatch?.context && (`?

Reply:
We obviously can. 😄